Cargo.toml

```diff
@@ -25,6 +25,9 @@ keywords = [
 rustc-serialize = "*"
 docopt = "0.6"
 docopt_macros = "0.6"
-notify = "^1.1.3"
 regex = "*"
 time = "*"
+
+[dependencies.notify]
```

Comment:
Don't do that. I'll crate up notify soon, if you want me to do that sooner remind me about it, but don't make this Cargo.toml depend on non- crates.io things, please.